pkg/products/monitoring/prometheusRules.go

```diff
@@ -292,6 +292,22 @@ func (r *Reconciler) newAlertsReconciler() resources.AlertReconciler {
 					},
 				},
 			},
+			{
+				AlertName: "install-upgrade-alerts",
+				Namespace: r.Config.GetOperatorNamespace(),
+				GroupName: "general.rules",
+				Rules: []monitoringv1.Rule{
+					{
+						Alert: "RHMICSVAbnormal",
+						Annotations: map[string]string{
+							"message": "An abnormal CSV was found, possibly preventing an operator from installing/upgrading",
```

Comment:
Don't know if this message conveys the information about what's happening sufficiently, thoughts?

Reply:
How about including some of the labels in the message?

e.g. `RequirementsNotMet for CSV '{{$labels.name}}' in namespace '{{$labels.namespace}}'. Phase is {{$labels.phase}}`